src/PathUtils.ts

```diff
@@ -44,6 +44,15 @@ export class PathUtils {
         }
     }
 
+    /**
+     * Returns true if the editor is in Edit Mode
+     *
+     * @returns {Boolean} the result of the check of the Editor mode
+     */
+    public static isEditMode(): boolean {
+        return this.getMetaPropertyValue(MetaProperty.WCM_MODE) === "edit" ? true : false;
```

Comment:
The meta property on a different domain cannot be set stateful on AEM backend via HTL. We discussed introducing a query parameter for supporting Apps hosted on a different domain. Please collaborate with @grubyak  on that very matter

Please keep the meta property as a fallback for cases where the content is hosted by AEM